source/Calamari.Tests/KubernetesFixtures/ApiResourceOutputParserTests.cs

```diff
@@ -0,0 +1,59 @@
+"podtemplates                                     v1                                true         PodTemplate                        create,delete,deletecollection,get,list,patch,update,watch   ",
+"replicationcontrollers              rc           v1                                true         ReplicationController              create,delete,deletecollection,get,list,patch,update,watch   all",
+"resourcequotas                      quota        v1                                true         ResourceQuota                      create,delete,deletecollection,get,list,patch,update,watch   ",
+"secrets                                          v1                                true         Secret                             create,delete,deletecollection,get,list,patch,update,watch   ",
+"serviceaccounts                     sa           v1                                true         ServiceAccount                     create,delete,deletecollection,get,list,patch,update,watch   ",
+"services                            svc          v1                                true         Service                            create,delete,deletecollection,get,list,patch,update,watch   all",
+"mutatingwebhookconfigurations                    admissionregistration.k8s.io/v1   false        MutatingWebhookConfiguration       create,delete,deletecollection,get,list,patch,update,watch   api-extensions",
+"validatingadmissionpolicies                      admissionregistration.k8s.io/v1   false        ValidatingAdmissionPolicy          create,delete,deletecollection,get,list,patch,update,watch   api-extensions",
+"validatingadmissionpolicybindings                admissionregistration.k8s.io/v1   false        ValidatingAdmissionPolicyBinding   create,delete,deletecollection,get,list,patch,update,watch   api-extensions",
+"validatingwebhookconfigurations                  admissionregistration.k8s.io/v1   false        ValidatingWebhookConfiguration     create,delete,deletecollection,get,list,patch,update,watch   api-extensions",
+"customresourcedefinitions           crd,crds     apiextensions.k8s.io/v1           false        CustomResourceDefinition           create,delete,deletecollection,get,list,patch,update,watch   api-extensions",
+"apiservices                                      apiregistration.k8s.io/v1         false        APIService                         create,delete,deletecollection,get,list,patch,update,watch   api-extensions",
+"controllerrevisions                              apps/v1                           true         ControllerRevision                 create,delete,deletecollection,get,list,patch,update,watch   "
+            };
+
+            var parsedResult = ApiResourceOutputParser.ParseKubectlApiResourceOutput(outputLines);
+
+            parsedResult.Should()
```

Comment:
test a bunch of the variations such as with/without shortnames, namespaced and different api versions